components/constellation/pipeline.rs

```diff
@@ -162,7 +162,7 @@ pub struct InitialPipelineState {
     pub mem_profiler_chan: profile_mem::ProfilerChan,
 
     /// Information about the initial window size.
```

Comment:
Should this doc comment be updated as well?

Reply:
Yep. I've updated this.